packages/grafana-ui/src/components/Modal/Modal.tsx

```diff
@@ -4,6 +4,8 @@ import { FocusScope } from '@react-aria/focus';
 import { OverlayContainer, useOverlay } from '@react-aria/overlays';
 import React, { PropsWithChildren, useRef } from 'react';
 
+import { t } from 'app/core/internationalization';
```

Comment:
```suggestion
import { t } from '../../utils/i18n';
```

in grafana-ui, we need to import the utils from a different place as per https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#how-to-add-a-new-translation-phrase:~:text=When%20translating%20in%20grafana%2Dui%2C%20use%20a%20relative%20path%20to%20import%20%3CTrans%20/%3E%20and%20t()%20from%20src/utils/i18n.